marquez_client/client.py

```diff
@@ -79,21 +80,24 @@ def list_namespaces(self, limit=None, offset=None):
             }
         )
 
-    def create_job(self, job_name, location, input_dataset_urns=None,
+    def create_job(self, job_name, job_type, location, input_dataset_urns=None,
                    output_dataset_urns=None, description=None,
                    namespace_name=None):
         if not job_name:
             raise ValueError('job_name must not be None')
+        if not job_type:
+            raise ValueError('job_type must not be None')
         if not location:
             raise ValueError('location must not be None')
 
         if not namespace_name:
             namespace_name = self._namespace_name
 
         payload = {
-            'inputDatasetUrns': input_dataset_urns or [],
-            'outputDatasetUrns': output_dataset_urns or [],
-            'location': location
+            'inputs': input_dataset_urns or [],
+            'outputs': output_dataset_urns or [],
```

Comment:
I'd rename the `_dataset_urns` part and remove the urn, as it might be confusing.

Reply:
resovled.